core/.js/src/main/scala/io/udash/bindings/inputs/Input.scala

```diff
@@ -7,26 +7,39 @@ import scalatags.JsDom.all._
 
 import scala.concurrent.duration.{Duration, DurationInt}
 
-/** Abstraction for HTML input tags.*/
+/** Abstraction for HTML input tags. */
 private[bindings] abstract class Input(inputType: String) {
   /**
-    * @param value Property to bind.
-    * @param debounce Property update timeout after input changes.
-    * @param inputModifiers Additional Modifiers, don't use modifiers on value, onchange and onkeyup attributes.
-    * @return HTML input with bound Property, applied modifiers and nested options.
-    */
-  def apply(value: Property[String], debounce: Duration = 20 millis)(inputModifiers: Modifier*): InputBinding[JSInput] =
+   * @param value               Property to bind.
+   * @param debounce            Property update timeout after input changes.
+   * @param onInputElementEvent Callback that's executed when `Input` element receives one of following events:
+   *                            `Input`, `Change`, `KeyUp`, `Paste` and element value is different than property value.
+   *                            Can be used to mimic unidirectional data flow for components based on this `Input`.
```

Comment:
nit: does it mimic it or is it actually unidirectional

Reply:
I mean, this component is still bidirectional(it has exposed property), but can be used to create unidirectional layer for higher level components